### PR TITLE
Couple of changes for OO import

### PR DESCRIPTION
**Pull request by jits**
Please see individual commits.

### CODE REVIEW

File: evaluate.py [hunk 1]
```diff
@@ -125,8 +127,12 @@ def pump_it(self):
 
         # TODO: validate the object
         try:
+            # Change the reader config if a source file was specified via args
+            if args.source_xml:
+                profile["pipeline"]["reader"][1]["file_path"] = args.source_xml
```
Comment (dchaplinsky) [inline evaluate.py]: That looks a bit dangerous as you are relying on an order of arguments here, so things can explode

Comment (jits) [reply evaluate.py]: Yep, indeed it is quite brittle, but I don't see another way unless we deserialise the YAML in to first class config objects.

Mind you, I can make this a little less brittle by finding the first entry that has a `file_path` and updating only that one – will make that change.

**Comment by jits** [comment]
@dchaplinsky – updated: https://github.com/openownership/ua-edr-extractor/pull/1/commits/30a376b8d6c6569a10290ef299eddd677aa9aee0

**Comment by jits** [comment]
Hi again @dchaplinsky – I've added in [the ability to override/set the limit via a command line arg](https://github.com/openownership/ua-edr-extractor/pull/1/commits/d18424716927fefebf567a5bb636ff40d5ab804f) – have a look whenever you have a sec :)

File: evaluate.py [hunk 2]
```diff
@@ -125,8 +129,14 @@ def pump_it(self):
 
         # TODO: validate the object
         try:
+            # Change the reader config if a source file was specified via args
+            if args.source_xml:
+                reader_file_path_entry = next((x for x in profile["pipeline"]["reader"] if isinstance(x, dict) and x.get("file_path")), None)
```
Comment (dchaplinsky) [inline evaluate.py:134]: Smells like a ruby :)

Comment (jits) [reply evaluate.py:134]: I was amazed I could do that in Python 😉

**Pull request merged**